### ISSUE TITLE
Minor - warnings on MacOS from audio.hpp and midi.hpp

### ISSUE THREAD

**Issue by baconpaul** [VCVRack/Rack#847]
Hi!

This is a minor point. When i do a clean build of rack on macOS I get a collection of warnings from files which include audio.hpp or midi.hpp.

```
include/audio.hpp:6:32: warning: unknown warning group '-Wsuggest-override', ignored [-Wunknown-warning-option]
#pragma GCC diagnostic ignored "-Wsuggest-override"
```

and

```
include/midi.hpp:10:32: warning: unknown warning group '-Wsuggest-override', ignored [-Wunknown-warning-option]
#pragma GCC diagnostic ignored "-Wsuggest-override"
```

These are pretty easy to squash. Just an ifndef ARCH_MAC around the diagnostic ignored, which I guess clang doesn't support

```
diff --git a/include/audio.hpp b/include/audio.hpp
index 972d4ec..c1f5aac 100644
--- a/include/audio.hpp
+++ b/include/audio.hpp
@@ -3,7 +3,9 @@
 #include <jansson.h>
 
 #pragma GCC diagnostic push
+#ifndef ARCH_MAC
 #pragma GCC diagnostic ignored "-Wsuggest-override"
+#endif
 #include <RtAudio.h>
 #pragma GCC diagnostic pop
````

If it helps I'm

```
paul:~/dev/VCVRack/Rack$ clang --version
Apple LLVM version 9.0.0 (clang-900.0.39.2)
Target: x86_64-apple-darwin17.4.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
paul:~/dev/VCVRack/Rack$ uname -a
Darwin PWLaptop.local 17.4.0 Darwin Kernel Version 17.4.0: Sun Dec 17 09:19:54 PST 2017; root:xnu-4570.41.2~1/RELEASE_X86_64 x86_64
```

With these ifdefs, there's only one warning in the whole rack build, which comes from nanovg.

This isn't hurting anything, of course, but hey, figured I'd ask.

Thanks for a great product - both the synth and dev environment - and a great community.

**Comment by AndrewBelt** [comment]
Fixed in 62bbf5ef461a6241069bad7981c0b409fdd38223

**Comment by baconpaul** [comment]
So great. Thanks!
